### PR TITLE
List dot tokens

### LINKED ISSUE #6: List DOT token asset
Documentation: https://github.com/MetaMask/metamask-snaps-beta/wiki/Snaps-API#custom-asset-management

- you should fetch balance for generated account from public node
- balance should be displayed in Metamask

### PR DESCRIPTION
**Pull request by MakMuftic**
Closes #6 

Waiting on the faucet for a proper test.

### CODE REVIEW

File: src/asset/unit.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import {getBalance} from "../rpc/substrate/getBalance";
+import {getAddress} from "../rpc/getAddress";
+import {executeAssetOperation} from "./executeAssetOperation";
+
+export function unitAsset(balance: string, address: string): Asset {
+  return {
+    balance: balance,
+    customViewUrl: `https://polkascan.io/pre/kusama/account/${address}`,
+    decimals: 0,
+    identifier: 'unit-test-asset',
```
Comment (mpetrunic) [inline src/asset/unit.ts]: ksm-asset

File: src/asset/unit.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    image: '',
+    symbol: 'KSM',
+  };
+}
+
+export async function manageUnitAsset(wallet: Wallet, api: ApiPromise, method: "update" | "add"): Promise<Asset> {
```
Comment (mpetrunic) [inline src/asset/unit.ts]: - rename this to createPolkadotAsset
- later on we will add onether method updateAssetBalance

File: src/asset/unit.ts [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import {Asset, Wallet} from "../interfaces";
+import ApiPromise from "@polkadot/api/promise";
+import {getBalance} from "../rpc/substrate/getBalance";
+import {getAddress} from "../rpc/getAddress";
+import {executeAssetOperation} from "./executeAssetOperation";
+
+export function unitAsset(balance: string, address: string): Asset {
```
Comment (mpetrunic) [inline src/asset/unit.ts]: getPolkadotAssetDescription

File: src/asset/unit.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  return {
+    balance: balance,
+    customViewUrl: `https://polkascan.io/pre/kusama/account/${address}`,
+    decimals: 0,
+    identifier: 'unit-test-asset',
+    image: '',
```
Comment (mpetrunic) [inline src/asset/unit.ts]: put this logo:
https://coinranking.com/coin/ePlOuwd_e+kusama-ksm

Comment (MakMuftic) [reply src/asset/unit.ts]: @mpetrunic I am not sure why the image is not showing. I checked their [examples repos](https://github.com/MetaMask/snaps-cli/tree/master/examples/custom-token) and don't see any problem with our image. I even scaled it to 200x200 (as in the example) and rehosted it but it won't show. I will leave it for now (left URL inside code but it is not working)

Comment (mpetrunic) [reply src/asset/unit.ts]: we'll figure it out

**Pull request merged**